packages/fscomponents/src/components/Button.tsx

```diff
@@ -147,4 +157,26 @@ export class Button extends PureComponent<ButtonProps, ButtonState> {
       );
     }
   }
+
+  get titleState(): string {
+    const {
+      title,
+      selectedTitleState,
+      dynamicTitleStates
+     } = this.props;
+
+    if (
+      (!selectedTitleState && selectedTitleState !== 0) ||
+      !dynamicTitleStates ||
+      (
+        selectedTitleState &&
+        dynamicTitleStates &&
```

Comment:
You shouldn't need these two checks because it won't reach this part if these are falsey (except for if selectedTitleState is zero, but that would also be fine). Also, selectedTitleState should only be a number or undefined, so you should be able to just check to see if it is undefined.

Reply:
@Cauldrath done